### ISSUE TITLE
Using esserializer with nextjs: Cannot read properties of undefined (reading 'name')

### ISSUE THREAD

**Issue by senser1** [shaochuancs/esserializer#105]
I'm having the same issue as mentioned in [issue 101](https://github.com/shaochuancs/esserializer/issues/101). I've tested with NextJS 12 and the new NextJS 13.

**Comment by mwcshao** [comment]
Hi, senser1

Thanks for your feedback. Does this error occur immediately after ESSerializer is required? Or does it occur when performing serialize/deserialize operation? Could you post this TypeError's error stack?

Thanks

Chuan

**Comment by senser1** [comment]
Importing ESSerializer on a NextJS app is sufficient to reproduce the issue: 

```import "../styles/globals.css";

import ESSerializer from "esserializer";

function MyApp({ Component, pageProps }) {
  return <Component {...pageProps} />;
}

export default MyApp;
```

Stack trace:

```index.js?46cb:601 Uncaught TypeError: Cannot read properties of undefined (reading 'name')
    at 607 (bundle.js?863d:1:389)
    at __nested_webpack_require_13848__ (bundle.js?863d:1:14002)
    at eval (bundle.js?863d:1:14097)
    at eval (bundle.js?863d:1:14122)
    at ./node_modules/esserializer/dist/bundle.js (_app.js?ts=1666851984736:17:1)
    at options.factory (webpack.js?ts=1666851984736:661:31)
    at __webpack_require__ (webpack.js?ts=1666851984736:37:33)
    at fn (webpack.js?ts=1666851984736:316:21)
    at eval (webpack-internal:///./pages/_app.js:6:70)
    at ./pages/_app.js (_app.js?ts=1666851984736:92:1)
    at options.factory (webpack.js?ts=1666851984736:661:31)
    at __webpack_require__ (webpack.js?ts=1666851984736:37:33)
    at fn (webpack.js?ts=1666851984736:316:21)
    at eval (?6782:5:16)
    at eval (route-loader.js?ea34:211:51)
```


I've tested the above by creating a brand new NextJS app (via `npx create-next-app@latest`). Hope this helps to troubleshoot the issue :)

**Comment by mwcshao** [comment]
Thank you senser1! I'll try to fix this issue next week.

**Comment by shaochuancs** [comment]
Bug fixed in v1.3.4
